vrs/vrs_annotation_batch.py

```diff
@@ -169,30 +174,34 @@ def main(args):
 
     batch_vrs = hb.Batch(name="vrs-annotation", backend=backend)
 
-    for vcf_index in file_list:
+    for vcf_name in file_list:
 
         # Setting up job
         new_job = init_job_with_gcloud(
             batch=batch_vrs,
-            name=f"VCF_job_{vcf_index.split('.')[0].split('-')[1]}",
+            name=f"VCF_job_{vcf_name.split('.')[0].split('-')[1]}",
             image=args.image,
             mount=working_bucket,
         )
 
-        # Script path for annotation is not expected to change for GA4GH if functionality is installed using pip in the DockerFile
-        # ...unless GA4GH team changes their file structure and a new Image is used
+        # Script path for annotation is not expected to change for GA4GH if dependencies are installed using pip in the DockerFile
         vrs_script_path = (
             "/usr/local/lib/python3.9/dist-packages/ga4gh/vrs/extras/vcf_annotation.py"
         )
-        # Print which file is being annotated, create directory for annotated shard, and then perform annotation
+
+        # Store VCF input path read as a Batch InputResourceFile
```

Comment:
is doesn't look like an InputResourceFile (https://hail.is/docs/batch/api/batch/hailtop.batch.batch.Batch.html?highlight=read_input#hailtop.batch.batch.Batch.read_input) is actually used anywhere yet - are there still planned changes?

Reply:
I tried to incorporate InputResourceFiles on Friday out of curiosity but it didn't work out - but I accidentally still left the comment in, my mistake!

Reply:
```suggestion
        # Store VCF input and output paths
```